tests/testflows/map_type/configs/clickhouse/config.d/storage.xml

```diff
@@ -0,0 +1,20 @@
+<yandex>
+
+<storage_configuration>
+    <disks>
+        <default>
+            <keep_free_space_bytes>1024</keep_free_space_bytes>
```

Comment:
Why do we need this to test Map data type?

Reply:
Removed.